src/util/styles/index.js

```diff
@@ -3,14 +3,28 @@
  */
 import { default as _isDarkColor } from 'is-dark-color'
 import { lowerFirst, clamp } from 'lodash'
+import rgba from 'color-rgba'
 
 /**
  * WordPress dependencies
  */
 import { sprintf } from '@wordpress/i18n'
 
 export const isDarkColor = color => {
```

Comment:
As with the `hexToRgba` function, the `isDarkColor` function should only perform checking whether the color is dark. Please keep all global color functions in the global color area.